e2e-tests/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
I haven't encountered an `.editorconfig` file before, is it kind of like an `.idea` analogue? If so then maybe it should be git-ignored.

Reply:
It should not be ignored. It's to ensure when editing code in this folder that your editor is using the proper text format settings: https://editorconfig.org/

Reply:
Oh cool, haven't seen that standard before!

Reply:
I will say it's not strictly required because I added Prettier to the project. But it's nice if you have Editorconfig in your editor already.

Reply:
There's also a `.editorconfig` already in the top-level repo; it was set up for the Java projects. I broadly agree with your comment [over here](https://github.com/NASA-AMMOS/aerie/pull/122#discussion_r840097019) regarding monorepos, but maybe it makes sense to have a single consistent `.editorconfig`? It's more about the human developers interacting with Aerie projects than the specifics of any one project.

If you agree, maybe it would be good to merge this in with the top-level `.editorconfig`? Alternatively, we could duplicate the current `.editorconfig` into all Java projects, but this seems like a recipe for accidentally having subtly different code styles throughout the codebase.